app_test.go

```diff
@@ -503,7 +503,6 @@ func TestApp_Float64Flag(t *testing.T) {
 }
 
 func TestApp_ParseSliceFlags(t *testing.T) {
-	var parsedOption, firstArg string
```

Comment:
the Go command complained that these were declared and not used

Reply:
These are not used but they do have side effects/assert evaluation logic so should probably be called?

Reply:
@AudriusButkevicius the code didn't even compile though, so im not sure how that's supposed to work? In any case, I can just add `var _ = parsedOption` below

Reply:
Its not the variable declarations, its the calls that assign to the unused variables is what matters.

Reply:
@AudriusButkevicius again, the code doesn't even compile and tests don't run :)

Reply:
This is because we're just re-assigning those variables, but not really doing anything with them. It could be that go1.8 (what travis is using) didn't care, but Go 1.13 is smarter.